demo/components/opt-in-flyout/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="/node_modules/@brightspace-ui/core/components/demo/styles.css" type="text/css">
+		<script type="module">
+			import '@brightspace-ui/core/components/demo/demo-page.js';
+			import '../../../src/components/opt-in-flyout/opt-in-flyout.js';
+			import '../../../src/components/opt-in-flyout/opt-out-flyout.js';
+			import '../../../src/components/opt-in-flyout/opt-out-reason.js';
+		</script>
+		<title>opt-in-flyout demo</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+		<style>
+			.flyout-wrapper {
+				height: 450px;
+				position: relative;
+				width: 100%;
+			}
+		</style>
+	</head>
+	<body>
```

Comment:
Do you want to include `unresolved` here. The `.../demo/styles.css` files contains the style to set... 
```css
body[unresolved] {
	opacity: 0;
}
```
When I was putting together my AI image description demo page I did notice I was seeing some FOUC without it. I think mostly resulting from the delay loading the font.

Reply:
Sure -- I'll add that to the create template as well.